mod/calculated_metrics/set/abstract/value/update_calculations.rb

```diff
@@ -0,0 +1,11 @@
+event :update_related_scores, :after_integrate, skip: :allowed do
+  ensure_metric(metric_card).each_dependent_score_metric do |metric|
+    metric.update_value_for! company: company_id, year: year
+  end
+end
+
+event :update_related_calculations, :after_integrate, skip: :allowed do
+  ensure_metric(metric_card).each_dependent_formula_metric do |metric|
+    metric.update_value_for! company: company_id, year: year
+  end
+end
```

Comment:
what's the rationale for separating this from the other events?

Reply:
it's about calculated metrics